src/components/Navbar.jsx

```diff
@@ -154,4 +166,75 @@ const Navbar = () => {
   );
 };
 
-export default Navbar;
\ No newline at end of file
+const Search = () => {
+  const { data, isLoading, isError } = useGetAllCategoriesQuery();
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    if (data) {
+      setCategories(data.data);
+    }
+  }, [data]);
+
+  const dispatch = useDispatch();
+
+  const [formData, setFormData] = useState({
+    name: 'Product',
+    categoryIds: 76,
+    price: 0,
+    size: 20,
+    page: 1
+  });
```

Comment:
Why are these data hardcoded?

Reply:
This is when the user clicks on the search button and there is no actual search.,
it's  data that comes by default when user click on search button